.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
there is a pre-set node ignore list from github maybe take it

Reply:
These are sufficient, the candidate may add some if he wants